torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1811,6 +2088,8 @@ def hint():
         #    if we have an input (2, 3), we must show s0*2 == 2 and s1 == 3.
         #    This does a lot of work: it covers duck sizing and equality guards.
         exprs = []
+        dim_constraints = DimConstraints(symbol_to_source, self.var_to_val)
```

Comment:
This work is unused if summarize_dim_constraints is False, right?

Reply:
Yeah. I thought about hiding it but it's not anything heavy that we don't do already (e.g., we're not calling any solvers when initializing).